plugins/newsdownloader.koplugin/epubdownloadbackend.lua

```diff
@@ -24,6 +24,54 @@ local max_redirects = 5; --prevent infinite redirects
 local TIMEOUT_CODE = "timeout" -- from socket.lua
 local MAXTIME_CODE = "maxtime reached" -- from sink_table_with_maxtime
 
+-- filter HTML using CSS selector
+local function filter(text, element)
+    local htmlparser = require("htmlparser")
```

Comment:
It's clearer to put this at the top on line 3.

Reply:
Isn't it better to not import the library if it is only used in case enable_filter=true? Or is the time needed for such import negligent?

Reply:
It's negligible, yes. But it can stay here if you want it here.